packages/web-components/src/components/ic-top-navigation/test/basic/ic-top-navigation.mobile.e2e.ts

```diff
@@ -77,6 +77,7 @@ describe("ic-top-navigation on mobile", () => {
       <ic-navigation-item
         label="Navigation"
         slot="navigation"
+        href="#"
```

Comment:
Is this line supposed to be here? If so, for my understanding what is the purpose of having a href="#"?